nconf/tls.go

```diff
@@ -21,88 +24,89 @@ type TLSConfig struct {
 }
 
 func (cfg TLSConfig) TLSConfig() (*tls.Config, error) {
-	var tlsconf *tls.Config
 	var err error
-	if cfg.Cert != "" && cfg.Key != "" {
-		tlsconf, err = LoadFromValues(cfg.Cert, cfg.Key, cfg.CA)
-	} else if cfg.CertFile != "" && cfg.KeyFile != "" {
-		tlsconf, err = LoadFromFiles(cfg.CertFile, cfg.KeyFile, cfg.CAFiles)
-	}
 
-	if err != nil {
-		return nil, err
+	tlsConf := &tls.Config{
+		MinVersion:         tls.VersionTLS12,
+		InsecureSkipVerify: cfg.Insecure,
 	}
 
-	if tlsconf != nil {
-		tlsconf.InsecureSkipVerify = cfg.Insecure
+	// Load CA
+	if cfg.CA != "" {
+		logrus.Debugf("Loading CA value")
```

Comment:
i wouldn't add a statement here

Reply:
Generally I don't like adding built in log statements to a library

Reply:
fair. I wanted to know if the CA/Certificate was loaded and from where. If NATS client fails to connect because the CA is missing, it won't hint you in that direction and it took me some testing and debugging to figure out that the env var was incorrect and the CA was not loaded